src/data/ingestion.json

```diff
@@ -388,11 +414,17 @@
         }
       },
       "patch": {
-        "tags": ["ingestion"],
+        "tags": [
```

Comment:
Is `authentications` more correct than `ingestion` here? 👀

Reply:
yes we are already in the `ingestion` "context" and `authentications` is a building block of the platform so with something like https://github.com/algolia/mcp-node/pull/44#discussion_r2096313499 that would be best